GlotPress/GlotPress-WP
# Merge the GlotDict hotkeys in GlotPress

## What?

This PR improves the GlotPress hotkeys, as we [discussed here](https://github.com/GlotPress/GlotPress/discussions/1415).

## Why?

In the last years, [GlotDict](https://github.com/Mte90/GlotDict) has added some interesting hotkeys for [translate.wordpress.org](https://translate.wordpress.org). This PR tries to bring the changes to GlotPress, so we can have the same new or updated hotkeys in GlotPress an at translate.w.org. Also, fixes some missing hotkeys for macOS.

## How?

This PR updates the hook for the keydown event, available in the editor.js file. This update only affects the editor.
This PR adds a new file, `assets/js/hotkeys.js` with the global hotkeys, working in all GlotPress functionality, not only in the editor.

## Testing Instructions




1. Press `?` to show the popup with the help, or click on the new link in the footer.
2. Press `Esc` to close the help.
3. In the help popup, you can see all the hotkeys available.

## Screenshots or screencast 

This is the new link in the footer. When you click it, you will see the popover.

![image](https://user-images.githubusercontent.com/1667814/175280852-b359f09e-34bf-41d2-9304-438832f7016b.png)

<img src="https://user-images.githubusercontent.com/1667814/175280918-2d9c2709-9747-45ac-8ccf-a4c6b6e0c774.png" width=400px>

This is the popover with a list of the hotkeys. This popover can be used as documentation of the hotkeys.

![image](https://user-images.githubusercontent.com/1667814/175009160-b5e57d5c-48b9-4c67-b302-a58e344ee236.png)

![image](https://user-images.githubusercontent.com/1667814/186658715-91345636-82c4-4ed6-a983-bdfb5cb18f17.png)
